Shared/Tracking/Tracker.swift

```diff
@@ -31,6 +31,8 @@ protocol Tracker: AnyObject {
     /// A boolean indicating if the tracker is currently logged in.
     var isLoggedIn: Bool { get }
 
+    var restrictSources: [String]? { get }
```

Comment:
add some documentation here
```suggestion
    /// An array of sources that this tracker can be used with
    var restrictSources: [String]? { get }
```

but I also don't think it should be optional. default value should just be an empty array.

Reply:
I thought nil could be more relevant for a non working filter, instead if this variable was meant for disabling a source, the empty array could be a better option.

Reply:
when would you have it be nil instead of an empty array? the convention in swift is to not have optional arrays, it's to use an empty array in the same way as you would nil

Reply:
I'm not experienced in swift, sorry. Now it's an empty array